docs/MAIL-ETIQUETTE

```diff
@@ -200,10 +200,10 @@ MAIL ETIQUETTE
 
   This is why top posting is so bad:
 
-      A: Because it messes up the order in which people normally read text.
       Q: Why is top-posting such a bad thing?
-      A: Top-posting.
+      A: Because it messes up the order in which people normally read text.
       Q: What is the most annoying thing in e-mail?
+      A: Top-posting.
```

Comment:
The point was to write the lines in top-posting order, which is reversed. When you unreverse them, that point is lost.  I think that is a pity.